src/app/modules/attribution/features/features.html

```diff
@@ -10,51 +10,46 @@ <h5>{{ config.label }}</h5>
         <ng-container [formGroup]="attribution.form">
           <ng-container formArrayName="presets">
             <ng-container *ngFor="let preset of attribution.form.controls.presets.controls; let i=index">
-              <ngb-accordion  #acc="ngbAccordion" destroyOnHide="false" [activeIds]="panelIds">
-                <ngb-panel class="feature-panel">
-                  <ng-template ngbPanelTitle>
-                    <div class="row">
-                      <div class="md-col-12">
-                        <div class="d-flex flex-row">
-                          <div class="mr-auto p-2">
-                            <ng-container
-                              feature
-                              class="features-header"
-                              field="name"
-                              type="light"
-                              [group]="attribution.form"
-                              [nestedGroupIndex]="i">
-                            </ng-container>
-                          </div>
-                          <div class="p-2">
-                            <button class="form-control btn btn-sm" type="button">
-                              <i class="fa fa-minus-square-o" *ngIf="acc.isExpanded('ngb-panel-' + i)"> </i>
-                              <i class="fa fa-plus-square-o" *ngIf="!acc.isExpanded('ngb-panel-' + i)"> </i>
-                            </button>
-                          </div>
-                          <div class="p-2">
-                            <button class="form-control btn btn-sm p-2" type="button" (click)="removeFeatureCard(i)">
-                              <i class="fa fa-trash-o"> </i>    
-                            </button>
-                          </div>
-                        </div>
-                      </div>
+              <div class="preset-card row">
+                <div class="col-md-12 preset-card-header">
+                  <div class="d-flex flex-row">
+                    <div class="mr-auto preset-features-header p-3">
+                      <ng-container feature class="features-header" field="name" type="light" [group]="attribution.form"
+                        [nestedGroupIndex]="i">
+                      </ng-container>
                     </div>
-                  </ng-template>
-                  <ng-template ngbPanelContent>
-                    <app-feature [config]=config [preset]=preset [i]=i></app-feature>
-                  </ng-template>
-                </ngb-panel>
-              </ngb-accordion>
+                    <div class="preset-accordion-button-container">
```

Comment:
by default can we make sure that the accordion is always open when a preset is first added?